### PR TITLE
Micnovak/parse log errors tests

### PR DESCRIPTION
**Pull request by micnovak**

### CODE REVIEW

**Review by jomiklos — COMMENTED** [review]
Tests are looking good imho - minus my two shortening proposals above.

I also propose to skip the big `.log` file with complex 69 problems parsing -> file size basically multiplies whole clean repository size.

File: flake8-check.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+set -e
+FLAKE8_ARGS="--count --max-line-length=120 --statistics"
+flake8 ${XMNR_DIR}/python ${XMNR_DIR}/test/unit/ ${FLAKE8_ARGS} --show-source
```
Comment (jomiklos) [inline flake8-check.sh:3]: maybe extra FLAKE8_CMD variable that has all the common parts of following two lines?

**Review by jomiklos — APPROVED** [review]

File: test/unit/test_parse_log_errors.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+from io import StringIO
+from typing import NamedTuple
+
+from drned_xmnr.op.parse_log_errors import ProblemsParser  # noqa
+
+
+class TestParseLog:
+
+    def test_parse_error_log(self):
+        # dataclass would be better, but not in python 3.6
```
Comment (martin-volf) [inline test/unit/test_parse_log_errors.py:13]: True... maybe we can stop supporting 3.6? It has been already EOL'd.

File: flake8-check.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+set -e
+FLAKE8_ARGS="--count --max-line-length=120 --statistics --exclude namespaces"
+FLAKE8_CMD="flake8 ${XMNR_DIR}/python ${XMNR_DIR}/test/unit/ ${FLAKE8_ARGS}"
```
Comment (martin-volf) [inline flake8-check.sh:3]: I think we can add drned-skeleton too.

**Review by martin-volf — APPROVED** [review]

**Pull request merged**